src/main/java/io/jenkins/plugins/oak9/utils/Severity.java

```diff
@@ -25,7 +26,7 @@
      * @return boolean to indicate if this passes or fails
      */
     public static boolean exceedsSeverity(int maxSeverity, String currentSeverity) {
-        return Severity.severities.get(currentSeverity.trim().toLowerCase()) >= maxSeverity;
+        return (Severity.severities.get(currentSeverity.trim().toLowerCase()) >= maxSeverity && maxSeverity > 0);
```

Comment:
Nitpicking here but you could make `maxSeverity > 0` the first evaluation to short-circuit out and avoid the severities lookup.